### PR TITLE
Add known CRYSTAL issue

### PR DESCRIPTION
**Pull request by kevinstratford**
I've added this CRYSTAL-specific issue to the CRYSTAL page.

### CODE REVIEW

File: docs/other-software/crystal.md [hunk 1]
```diff
@@ -52,3 +52,19 @@ cp tio2.d12 INPUT
...
+
+Long-running jobs may encounter unexpected errors of the form
+```
+slurmstepd: error: Detected 1 oom-kill event(s) in step 411502.0 cgroup.
+```
+These are related to a memory leak in the underlying libfabric communication
+layer, which will be fixed in a future release. In the meantime, it should
+be possible to work around the problem by adding
+```
+export FI_MR_CACHE_MAX_COUNT=0
```
Comment (jsindt) [inline docs/other-software/crystal.md:68]: Please could you confirm that this variable name is correct? Given that this is an OFI error, I want to make sure it shouldn't be `OFI_MR_CACHE_MAX_COUNT`.

**Review by jsindt — COMMENTED** [review]
@kevinstratford Please could you confirm that the variable name is correct? Once you've confirmed, I'm happy to approve.

File: docs/other-software/crystal.md (hunk 1 above)
Comment (kevinstratford) [reply docs/other-software/crystal.md:68]: `FI_MR_CACHE_MAX_COUNT` is correct.

**Review by jsindt — APPROVED** [review]
Thanks for confirming -- I'll approve.

**Pull request merged**